qdrant_client/grpc/__init__.py

```diff
@@ -5,3 +5,26 @@
 from .collections_service_pb2_grpc import *
 from .points_service_pb2_grpc import *
 from .snapshots_service_pb2_grpc import *
+
+
+# Compression Enum Implementation
+
+import enum
+import grpc
+
+@enum.unique
+class Compression(enum.IntEnum):
```

Comment:
I don't think we need a separate structure just to exclude `Deflate`, can we just use `isinstance` in init or deeper?

Reply:
@joein 
`grpc` module in `qdrant_client` does not support `Compression` enum
```py
from qdrant_client import grpc

grpc.Compression
# AttributeError: module 'qdrant_client.grpc' has no attribute 'Compression'. Did you mean: 'CompressionRatio'?
```

Thus we would either need to explicitly import Compression in `grpc/__init__.py` or keep the `grpc_compression` input as string. What would you suggest?